packer/Makefile

```diff
@@ -58,7 +65,7 @@ ifneq (, $(findstring -DPACK_LK, $(EVMM_CMPL_FLAGS)))
 	if [ $(LKBIN_DIR) ]; then cp $(LKBIN_DIR)lk.bin $(BUILD_DIR); fi
 else
 ifneq (, $(findstring -DPACK_OPTEE, $(EVMM_CMPL_FLAGS)))
-	if [ $(OPTEEBIN_DIR) ]; then cp $(OPTEEBIN_DIR)tee-pager.bin $(BUILD_DIR); fi
+	if [ $(OPTEEBIN_DIR) ]; then cp $(OPTEEBIN_DIR)tee-pager_v2.bin $(BUILD_DIR); fi
```

Comment:
why v2? it is better not to add version in a name of a binary, especially there's only 1 binary.

Reply:
There is no tee-pager.bin in optee-3.13, it is replaced by tee-pager_v2.bin